src/lib/multicall/Multicall.ts

```diff
@@ -372,10 +332,6 @@ export class Multicall {
       isAlchemy: isFallbackMode,
     });
 
```

Comment:
also thinking if we should add tests for this file, it doesn't seem to complex but quite frigile and important

Reply:
Yes, it sounds very reasonable. But I think I'll do it a little later, because now I have a feeling that this logic can change many times in the near future.

As an example, we are doing back and forth with this pr now: https://github.com/gmx-io/gmx-interface/pull/1237